.github/workflows/publish-cargo-crates.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish cargo crates
+
+on:
+  release:
+    types:
+      - released
+      - prereleased
+
+permissions:
+  packages: write
+  pull-requests: write
+  id-token: write
+  contents: read
+
+jobs:
+  publish-crates:
+    strategy:
+      matrix:
+        include:
+          - toml_file: "./scrypto-derive/Cargo.toml" 
+          - toml_file: "./radix-substate-store-queries/Cargo.toml" 
+          - toml_file: "./radix-common-derive/Cargo.toml"
+          - toml_file: "./radix-engine-profiling/Cargo.toml"
+          - toml_file: "./radix-substate-store-impls/Cargo.toml"
+          - toml_file: "./radix-clis/Cargo.toml"
+          - toml_file: "./radix-sbor-derive/Cargo.toml"
+          - toml_file: "./sbor-derive/Cargo.toml"
+          - toml_file: "./scrypto/Cargo.toml"
+          - toml_file: "./scrypto-test/Cargo.toml"
+          - toml_file: "./radix-transactions/Cargo.toml"
+          - toml_file: "./radix-native-sdk/Cargo.toml"
+          - toml_file: "./radix-blueprint-schema-init/Cargo.toml"
+          - toml_file: "./scrypto-compiler/Cargo.toml"
+          - toml_file: "./sbor-tests/Cargo.toml"
+          - toml_file: "./radix-engine-interface/Cargo.toml"
+          - toml_file: "./radix-rust/Cargo.toml"
+          - toml_file: "./sbor-derive-common/Cargo.toml"
+          - toml_file: "./radix-engine-profiling-derive/Cargo.toml"
+          - toml_file: "./radix-common/Cargo.toml"
+          - toml_file: "./sbor/Cargo.toml"
+          - toml_file: "./scrypto-derive-tests/Cargo.toml"
+          - toml_file: "./radix-engine/Cargo.toml"
+          - toml_file: "./radix-transaction-scenarios/Cargo.toml"
+          - toml_file: "./radix-engine-monkey-tests/Cargo.toml"
+          - toml_file: "./radix-engine-tests/Cargo.toml"
+          - toml_file: "./radix-substate-store-interface/Cargo.toml"
```

Comment:
I don't fully understand how matrics work, but we need to publish the crates sequentially in [this order](https://radixdlt.atlassian.net/wiki/spaces/S/pages/3367272451/Crate+Publishing).